packages/engine/src/scene/systems/LoadingSystem.ts

```diff
@@ -0,0 +1,90 @@
+import { AmbientLight, Box3, PerspectiveCamera, Scene, WebGLRenderer } from 'three'
+import { GLTFLoader } from '@xrengine/engine/src/assets/loaders/gltf/GLTFLoader'
+import { EngineEvents } from '@xrengine/engine/src/ecs/classes/EngineEvents'
+import { World } from '../../ecs/classes/World'
+import { System } from '../../ecs/classes/System'
+
+/**
+ * @author Dhara Patel <github.com/frozencrystal>
+ */
+
+export class LoadingRenderer {
+  static instance: LoadingRenderer
+  canvas: HTMLCanvasElement
+  scene: Scene
+  camera: PerspectiveCamera
+  renderer: WebGLRenderer
+  gltfScene: any
+  needsResize: boolean
+  constructor() {
+    LoadingRenderer.instance = this
+    //Todo
+    this.onResize = this.onResize.bind(this)
+
+    const loaderRendererCanvasId = 'loader-renderer-canvas'
+    const canvasElement = document.getElementById(loaderRendererCanvasId) as any
+    const url = 'https://172.160.10.156:3000/models/devices/razer_laptop.glb'
```

Comment:
@frozencrystal  Why is this hard coded ? can you please change

Reply:
@kinjalravel I believe this is a temporary solution - this will not be the model we use when this is fully implemented

Reply:
@frozencrystal @HexaField How about we pass this as props?